Non standard properties FAQ (#771)
rudderlabs/rudderstack-docs: docs/destinations/warehouse-destinations/faq.mdx

If the above steps take too long to complete, you can halt the warehouse operations in the interim. For more information, refer to this <Link to="#how-do-i-avoid-lags-in-data-syncs-while-heavy-loads-are-running-on-my-datawarehouse">FAQ</Link>.
</div>

### Why am I not able to see the properties added at the top level of an event in warehouse destination?

> RudderStack drops the non-standard properties (any property apart from the <Link to="/destinations/warehouse-destinations/warehouse-schema/#standard-rudderstack-properties">standard properties</Link>) declared at the top level of an event. However, it allows you to add such properties in the `context` or `properties` section of the event payload.

Reviewer: ```suggestion
RudderStack drops the non-standard properties (any property apart from the <Link to="/destinations/warehouse-destinations/warehouse-schema/#standard-rudderstack-properties">standard properties</Link>) declared at the top level of an event. However, you can add such properties in the `context` or `properties` section of the event payload.
```
